apps/src/lib/levelbuilder/script-editor/ScriptEditor.jsx

```diff
@@ -373,6 +383,15 @@ export default class ScriptEditor extends React.Component {
                     </p>
                   </HelpTip>
                 )}
+                {!this.props.hasCourse && (
+                  <HelpTip>
+                    <p>
+                      The family name is associate scripts that are standalone
+                      courses so that users can be redirected between different
+                      version years.
```

Comment:
sorry, I am not quite sure how to fix the wording here, but it seems like the wording is a little of at the beginning of this sentence.